operator/v1/types.go

```diff
@@ -70,11 +70,11 @@ type OperatorSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
 
-	// unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override
-	// it will end up overlaying in the following order:
-	// 1. hardcoded defaults
-	// 2. observedConfig
-	// 3. unsupportedConfigOverrides
+	// You can use `unsupportedConfigOverrides` to add configuration changes that are not supported by an operator. 
+	// Consider seeking guidance from the Red Hat support team before you use this property, 
+	// because using this property without knowledge of its capabilities could lead to unexpected behavior or conflict with other configuration options.
+	//
+	// Additionally, using this property blocks cluster upgrades, so remove this setting before you upgrade your cluster.
```

Comment:
Do we allow people to remove this value? Would have to find an SME for this as I don't actually know

Reply:
Thanks, @JoelSpeed . I'll ask an SME to confirm.

Reply:
Yes, the field can be removed.

Reply:
Thanks, @sanchezl and @JoelSpeed . I see an `optional` flag with this property, so I do not need to explicitly state that one can remove the property if they so wish?